docs/release-notes/5.7.40-43.md

```diff
@@ -58,6 +58,10 @@ Find the full list of bug fixes and changes in the [MySQL 5.7.40 Release Notes](
 * [PS-1098](https://jira.percona.com/browse/PS-1098): Manually rotating the log files by calling `audit_log_flush` multiple times caused incorrect file rotation and the wrong log file to be used.
 * [PS-8327](https://jira.percona.com/browse/PS-8327): `ALTER TABLE ... CHECK PARTITION` inside the stored procedure caused a server exit.
 
+## Platform support
+
+Percona Server for MySQL 5.7.40-43 does not support Ubuntu 22.04. Since Percona Server for MySQL 5.7 reaches EOL in October 2023, the support for Ubuntu 22.04 won't be added for Percona Server for MySQL 5.7 in future releases.
```

Comment:
We may not need the clarifying sentence. This section is usually we do, or we don't.

Reply:
Updated